### DISCUSSION TITLE
Consume Mode as a Config Option

### DISCUSSION THREAD

**Discussion by Kriegbaum** [MusicPlayerDaemon/MPD#2081] [Q&A] [answered]
I cruised around the discussions and documentation and I haven't seen this mentioned anywhere. Apologies if I've missed something:
Is there a way to set the default queue consume mode on MPD startup, like maybe a config file item? Consume  is my preferred operating mode and its kind of annoying to have to re-enable it every time I restart the server.

**Comment by owl770** [comment, marked as answer]
@Kriegbaum - if you have the state_file defined (and it gets saved correctly), consume mode should be restored on startup (along with other settings, like repeat, random, single).

**Reply by seamusdemora** [reply]
I've got a more basic question: What exactly is the "consume" mode? i.e. what does it control, or what difference does it make if it's ON or OFF?

**Reply by borine** [reply]
> What exactly is the "consume" mode?

RTFM - https://mpd.readthedocs.io/en/latest/protocol.html#playback-options

**Reply by Kriegbaum** [reply]
That worked thank you!
